src/voice-broadcast/stores/VoiceBroadcastPreRecordingStore.ts

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { TypedEventEmitter } from "matrix-js-sdk/src/models/typed-event-emitter";
+
+import { VoiceBroadcastPreRecording } from "..";
+import { IDestroyable } from "../../utils/IDestroyable";
+
+export type VoiceBroadcastPreRecordingEvent = "changed";
```

Comment:
Why not make this an enum?

Reply:
There was a discussion about this in web chapter. The conclusion was to prefer union types… One argument was, that you don't have to import the module if you want to use the value.